kubernetes/coturn.yml.erb

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+data:
+  # see https://github.com/matrix-org/synapse/blob/master/docs/turn-howto.md#configuration
+  turnserver.conf: |
+    no-tcp-relay
+
+    denied-peer-ip=10.0.0.0-10.255.255.255
+    denied-peer-ip=192.168.0.0-192.168.255.255
+    denied-peer-ip=172.16.0.0-172.31.255.255
```

Comment:
we may want to add 169.229.226.0/24, as connecting from a kubernetes workers may give special firewall access or privilege inside our network. we should keep in mind that TURN should work from desktops. i'm not sure what exactly this config does so i'm not exactly sure what to do here

Reply:
This option blacklists the IPs that the TURN server can holepunch. It should disallow connecting to anything behind a firewall/NAT that the TURN server is behind. I'll add `169.229.226.0/24`.